### PR TITLE
VX CTAs

### LINKED ISSUE #5469: Crosslink vx services
E.g., after uploading a dataset, one could present call-to-actions for aligning/segmenting a dataset. This flow could lead to an (automated?) email quite quickly to make it easier to get in touch with us.

There are probably several places in the wk UI and on the landing page where it would make sense to advertise the vx services.

### PR DESCRIPTION
**Pull request by philippotto**
### URL of deployed dev instance (used for testing):
- https://vxreferral.webknossos.xyz/datasets/upload

### Steps to test:
- open `datasets/upload`
- reload the page to rerun the random picking of the banner

### Issues:
- fixes #5469 

------
- [ ] Updated [(unreleased) changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)
- [ ] Updated [(unreleased) migration guide](../blob/master/MIGRATIONS.unreleased.md#unreleased) if applicable
- [ ] Updated [documentation](../blob/master/docs) if applicable
- [ ] Adapted [wk-connect](https://github.com/scalableminds/webknossos-connect) if datastore API changes
- [ ] Needs datastore update after deployment
- [X] Ready for review

### CODE REVIEW

File: frontend/javascripts/admin/dataset/dataset_add_view.js [hunk 1]
```diff
@@ -192,6 +201,72 @@ const DatasetAddView = ({ history, activeUser }: PropsWithRouter) => {
   );
 };
 
+function VoxelyticsBanner() {
+  const [showSegmentationBanner] = useState(Math.random() > 0.5);
+
+  // if (!features().isDemoInstance) {
```
Comment (philippotto) [inline frontend/javascripts/admin/dataset/dataset_add_view.js:207]: needs to be uncommented before merging

File: frontend/javascripts/types/api_flow_types.js [hunk 2]
```diff
@@ -539,7 +539,6 @@ export type APIFeatureToggles = {
   +discussionBoardRequiresAdmin: boolean,
   +addForeignDataset: boolean,
   +hideNavbarLogin: boolean,
-  +isDemoInstance: boolean,
```
Comment (philippotto) [inline frontend/javascripts/types/api_flow_types.js:542]: this entry appeared twice

**Comment by philippotto** [comment]
@hotzenklotz do want to have a look at https://vxreferral.webknossos.xyz/datasets/upload ? :)

**Comment by hotzenklotz** [comment]
LGTM. I think this is a good start. Let's get it out :-)

For easy reference:
image
Screenshot 2021-07-21 at 13 08 57

**Review by hotzenklotz — APPROVED** [review]
LGTM

**Pull request merged**